appcues/src/main/java/com/appcues/analytics/AutoPropertyDecorator.kt

```diff
@@ -100,7 +101,7 @@ internal class AutoPropertyDecorator(
             sessionPageviews += 1
         } else if (event.name == AnalyticsEvent.SessionStarted.eventName) {
             // on session start set _device key
-            attributes[DEVICE_PROPERTY] = hashMapOf<String, Any>().apply {
+            attributes[DEVICE_PROPERTY] = hashMapOf<String, Any?>().apply {
```

Comment:
just double checking - this allows the JSON serializing to send an explicit `null` value for `_pushToken`, when unset, correct?

Reply:
yeah good catch, i was only testing on the Recents Events, but for it to carry over to the network call we need to make sure Moshi does serialize it. the simplest one seems to be adding the @SerializeNull to EventRequest attributes, open to ideas.

Reply:
<img width="319" alt="image" src="https://github.com/appcues/appcues-android-sdk/assets/5244805/d1c4bdfb-9edf-4706-b7a3-2d9883d2a9ec">